spec/rubocop/cop/layout/space_around_operators_spec.rb

```diff
@@ -958,6 +958,22 @@ class ShowSourceTestClass < ShowSourceTestSuperClass
       RUBY
     end
 
+    it 'registers an offense when operator is followed by aligned << inside a string' do
+      expect_offense(<<~RUBY)
+        x   += foo
+            ^^ Operator `+=` should be surrounded by a single space.
+        'yz << bar'
+      RUBY
```

Comment:
Can you add `expect_correction`?

Reply:
Yes of course. Both tests are updated now.